spec/search_builders/curation_concerns/admin_set_search_builder_spec.rb

```diff
@@ -9,7 +9,7 @@
     before { builder.filter_models(solr_params) }
 
     it 'adds AdminSet to query' do
-      expect(solr_params[:fq].first).to include('{!field f=has_model_ssim}AdminSet')
```

Comment:
This is occurring because I removed unnecessary overrides which were using `field` rather than `terms`